dashboard-widgets.el

```diff
@@ -508,7 +507,7 @@ date part is considered."
    register-alist
    list-size
    "e"
-   (lambda (&rest ignore) (jump-to-register (car el)))
+   (lambda (&rest _ignore) (jump-to-register (car el)))
```

Comment:
Why no `'(lambda (&rest ignore) (jump-to-register (car el)))` as in the rest of sections?

Reply:
Yeah, others should be changed to `_ignore` as well to clean the warnings.